source/back.tex

```diff
@@ -158,6 +158,15 @@
 \printindex[conceptindex]
 }
 
+\clearpage
+\renewcommand{\preindexhook}{The bold page number for each entry is the page
+where the macro is defined.
+Other page numbers refer to pages where the macro is mentioned in the general text.}
```

Comment:
Needs a trait \\ before the closing brace, to ensure vertical space before the first item in the index.